VAPI_INTEGRATION_GUIDE.md

```
@@ -0,0 +1,538 @@
+  "objectName": "Contact"
+}
+
+// Test Search Objects
+{
+  "searchPattern": "Account"
+}
+```
+
+### **2. Integration Testing Scenarios**
+
+**Scenario 1: Recent Records**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)